packages/react-storage/src/components/StorageBrowser/actions/types.ts

```diff
@@ -3,7 +3,7 @@ import { LocationCredentialsProvider } from '../storage-internal';
 import { ActionState } from '../context/actions/createActionStateContext';
 
 export interface ActionInputConfig {
-  accountId: string;
+  accountId?: string;
```

Comment:
why change to optional? i thought this is required in storage browser 🤔

Reply:
This was done to avoid introducing a new prop & API change on the Amplify-auth StorageBrowser component. Discussed with Caleb, we only need the account ID for the control-plane APIs and will address that with the provider refactoring.